test/jdk/java/io/pathNames/win32/SJIS.java

```diff
@@ -46,7 +47,6 @@ public static void main(String[] args) throws Exception {
 
         /* This test is only valid on win32 systems
            that use the SJIS encoding */
-        if (File.separatorChar != '\\') return;
         String enc = System.getProperty("file.encoding");
         if ((enc == null) || !enc.equals("SJIS")) return;
```

Comment:
I just wonder this test has ever run since JDK18, as file.encoding is always UTF-8 unless COMPAT is specified (and this test case does not specify it)

Reply:
So do you think that this test should be removed? If so, I think that would be a different PR. Thanks.

Reply:
IIUC, the test tries to create double byte path directory, so in recent jdk it should check `sun.jnu.encoding==MS932` (it still only run on Japanese Windows though)

Reply:
Sounds like a different issue needs to be filed.

Reply:
Yeah, that sounds better.

Reply:
See https://bugs.openjdk.org/browse/JDK-8355558